### DISCUSSION TITLE
The error overlay should show the component stack trace instead of JS

### DISCUSSION THREAD

**Discussion by Timer** [vercel/next.js#62357] [Ideas]
Current:
![image](https://user-images.githubusercontent.com/616428/84943797-e4f2b500-b0b2-11ea-92d3-3c3e7e0030a1.png)

Desired:
![image](https://user-images.githubusercontent.com/616428/84943830-f50a9480-b0b2-11ea-8ec4-135b7189f30d.png)
![image](https://user-images.githubusercontent.com/616428/84943862-fdfb6600-b0b2-11ea-8c95-7ab4666e7692.png)

**Comment by samcx** [comment]
Moving this to <picture data-single-emoji=":nextjs:" title=":nextjs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/nextjs/2a85d633ae594556.png" alt=":nextjs:" align="absmiddle"></picture> [discussions](https://github.com/vercel/next.js/discussions) since this is a feature request!
